pkgs/top-level/all-packages.nix

```
@@ -10333,6 +10333,9 @@ with pkgs;
 
   smart-wallpaper = callPackage ../tools/X11/smart-wallpaper { };
 
+  nwaku = callPackage ../tools/security/nwaku { };
+  nim-waku = nwaku;
```

```suggestion
  nwaku = callPackage ../tools/security/nwaku { };
```
Why do we need this alias?

---

Because that's the old name of the project which is still in usage here and there.